rpcs3/Emu/Cell/Modules/cellMic.cpp

```diff
@@ -467,8 +1008,8 @@ s32 cellMicGetDeviceIdentifier()
 	return CELL_OK;
 }
 
-DECLARE(ppu_module_manager::cellMic)("cellMic", []()
-{
+DECLARE(ppu_module_manager::cellMic)
```

Comment:
the old style was OK, this also doesnt have curly brace on new line.

Reply:
Pretty sure it's the result of vs code formatting according to our clang format, I'll leave it as is or I'd have to remember to change it back after every formatting.